lib/urbanopt/scenario/default_reports/program.rb

```diff
@@ -75,6 +128,8 @@ def add_program(other)
           @conditioned_area += other.conditioned_area
           @unconditioned_area += other.unconditioned_area
           @footprint_area += other.footprint_area
+          @maximum_roof_height += other.maximum_roof_height
```

Comment:
Should be the max of the two, in Ruby that is:

`@maximum_roof_height += [@maximum_roof_height, other.maximum_roof_height].max`